gramps/gen/filters/rules/family/_regexpfathername.py

```diff
@@ -50,5 +58,13 @@ class RegExpFatherName(RegExpName):
         "matching a specified regular expression"
     )
     category = _("Father filters")
-    base_class = RegExpName
-    apply = father_base
+
+    def apply_to_one(self, db: Database, family: Family) -> bool:  # type: ignore[override]
+        father_handle = family.father_handle
+        if father_handle:
+            father = db.get_person_from_handle(father_handle)
+            if father:
+                return super().apply_to_one(db, father)
+            else:
+                return False
```

Comment:
```suggestion
```
Unnecessary else.

Reply:
I think it is more explicit, and "Explicit is better than implicit."